torch/onnx/utils.py

```diff
@@ -281,15 +304,14 @@ def _trace_and_get_graph_from_model(model, args, training):
     # before and after running the model.  Fail fast!
     orig_state_dict_keys = _unique_state_dict(model).keys()
 
-    # By default, training=False, which is good because running a model in
+    # By default, training=TrainingMode.EVAL, which is good because running a model in
     # training mode could result in internal buffers getting updated, dropout
     # getting applied, etc.  If you really know what you're doing, you
-    # can turn training=True (or None, to preserve whatever the original
-    # training mode was.)
-    with set_training(model, training):
```

Comment:
We are removing `set_training` here. This makes `training` in the method an unused parameter. Should it be remove from this method?